lib/components/breadcrumbs.vue

```diff
@@ -23,19 +23,16 @@ except according to the terms contained in the LICENSE file.
 
 <script>
 export default {
+  name: 'Breadcrumbs',
   props: {
-    breadcrumbs: {
+    list: {
       type: Array,
       required: true
     }
   },
   methods: {
     linkBreadcrumb(index) {
-      const breadcrumb = this.breadcrumbs[index];
-      return breadcrumb.view != null && index != this.breadcrumbs.length - 1;
-    },
-    view(breadcrumb) {
-      this.$emit('view', breadcrumb.view, breadcrumb.props);
+      return this.list[index].to != null && index !== this.list.length - 1;
```

Comment:
Is there not a way to get it to give you some kind of rich data object instead of mucking about with array indices?